### PR TITLE
DOCS-1870: Remove outdated Cloudflare Workers information from JavaScript driver docs

### PR DESCRIPTION
**Pull request by steve-fauna**

### CODE REVIEW

File: concourse/scripts/publish.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ then
   npm run build:package
   echo "Publishing a new version..."
   echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc
-  npm publish
+  npm publish --tag beta
```
Comment (faunaee) [inline concourse/scripts/publish.sh:20]: Is this change necessary?

Comment (steve-fauna) [reply concourse/scripts/publish.sh:20]: It was in the `v5` branch that I rebased from. Can one of the other reviewers confirm that it belongs there?

Comment (faunaee) [reply concourse/scripts/publish.sh:20]: If it was in the `v5` branch, it shouldn't be in your PR. You're making a documentation-only change, so the only modifications should be to the documentation.

Comment (steve-fauna) [reply concourse/scripts/publish.sh]: OK, change to publish.sh is undone.

**Review by faunaee — APPROVED** [review]
LGTM! 👍
